### PR TITLE
Add easy ways to test the entilement

### PR DESCRIPTION
**Pull request by kpouget**
```
toolbox/entitlement/test_in_podman.sh /path/to/key.pem
toolbox/entitlement/test_in_cluster.sh /path/to/key.pem
```

### CODE REVIEW

**Comment by omertuc** [comment]
A lot of the code created in this PR also shows up in [this PR](https://github.com/openshift-psap/ci-artifacts/pull/124), as if you accidentally branched one of the other rather than off master

**Comment by kpouget** [comment]
> A lot of the code created in this PR also shows up in [this PR](https://github.com/openshift-psap/ci-artifacts/pull/124), as if you accidentally branched one of off the other rather than off of master

indeed, thanks for point it out @omertuc, one commit was duplicated in #124, I removed it there

File: roles/entitlement_deploy/tasks/main.yml [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-  when: entitlement_pem | default('', true) | trim != ''
+- name: Compute the md5sum of the entitlement files (debug)
+  command: md5sum '{{ entitlement_rhsm }}' '{{ entitlement_pem }}'
+
+- name: "Deploy RHSM from file '{{ entitlement_rhsm }}'"
+  shell:
+    set -o pipefail;
+    python3 "{{ entitlement_py_apply }}"
+            "{{ entitlement_mc_rhsm }}" BASE64_ENCODED_RHSM_FILE "{{ entitlement_rhsm }}"
+    | oc create -f-
```
Comment (omertuc) [inline roles/entitlement_deploy/tasks/main.yml:22]: The other PR changed it from `create` to `apply` and now this PR is changing it back from `apply` to `create`

File: roles/entitlement_deploy/tasks/main.yml [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
+
+- name: "Deploy the pem and key-pem from file '{{ entitlement_pem }}'"
+  shell:
+    set -o pipefail;
+    python3 "{{ entitlement_py_apply }}"
+            "{{ entitlement_mc_pem }}" BASE64_ENCODED_PEM_FILE "{{ entitlement_pem }}"
+    | oc create -f-
```
Comment (omertuc) [inline roles/entitlement_deploy/tasks/main.yml:29]: Same here

File: roles/entitlement_deploy/tasks/main.yml (hunk 1 above)
Comment (kpouget) [reply roles/entitlement_deploy/tasks/main.yml]: good catch, fixed

File: roles/entitlement_test_wait_deployment/files/entitlement-tester_entrypoint.yml [hunk 3]
```diff
@@ -9,23 +9,39 @@ data:
...
+    echo
     echo "# Host OS Release"
     cat /etc-host/os-release
     source /etc-host/os-release
-    echo
 
-    echo "# md5sum of entitlement file (debug)"
-    if md5sum /etc/rhsm-host/rhsm.conf /etc/pki/entitlement-host/entitlement{,-key}.pem; then
-      echo "# INFO: entitlement files found"
+    if [[ "${REMOVE_HOST_ENTITLEMENT:-}" == "1" ]]; then
```
[the remark this answers is not in the document]
Comment (kpouget) [reply roles/entitlement_test_wait_deployment/files/entitlement-tester_entrypoint.yml:21]: sure

[the remark this answers is not in the document]
Comment (kpouget) [reply roles/entitlement_test_wait_deployment/files/entitlement-tester_entrypoint.yml:21]: fixed

**Pull request merged**